src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -62,6 +62,14 @@ namespace AppInstaller::Filesystem
         return (GetVolumeInformationFlags(path) & FILE_SUPPORTS_REPARSE_POINTS) != 0;
     }
 
+    bool PathEscapesBaseDirectory(const std::filesystem::path& path, const std::filesystem::path& base)
+    {
+        // Input for path must exist or this will throw a filesystem error.
+        const auto& pathString = std::filesystem::canonical(path).u8string();

```

Comment:
```suggestion
        const auto& pathString = std::filesystem::weakly_canonical(path).u8string();
```
Using `weakly_canonical` would make this more usable in the abstract case.  It could also be `lexically_normal`, which doesn't even care about whether any files exist or not.  But that doesn't prevent the potential for symlinks to enable escaping the location (although I don't know if it is possible to have a symlink in an archive file...).